int32 id

# Contains everything needed to reconstruct a Sample object
DataType[] sensor_data
